README.md

```diff
@@ -26,6 +26,7 @@ ______________________________________________________________________
 <details open>
 <summary><b>2024</b></summary>
 
+- \[2024/07\] Support InternVL2 full-serie models, InternLM-XComposer2.5 and function call of InternLM2.5
```

Comment:
I would recommend add a link to the to xcomposer2d5 https://github.com/InternLM/lmdeploy/blob/main/docs/zh_cn/multi_modal/xcomposer2d5.md

Reply:
Sure